lib/iris/unit.py

```diff
@@ -1823,9 +1823,9 @@ def convert(self, value, other, ctype=FLOAT64):
             >>> import numpy as np
             >>> c = unit.Unit('deg_c')
             >>> f = unit.Unit('deg_f')
-            >>> print(c.convert(0, f))
+            >>> print(np.round(c.convert(0, f), 8))
             32.0
-            >>> c.convert(0, f, unit.FLOAT32)
+            >>> np.round(c.convert(0, f, unit.FLOAT32), 8)
```

Comment:
You're right; this one is fine. I added it only to show some consistency with the 64-bit case.